tests/reg_tests/test_DVGeometry.py

```diff
@@ -1048,6 +1048,128 @@ def test_spanDV_child(self, train=False):
 
         np.testing.assert_allclose(dIdx["span"], dIdx_FD["span"], atol=1e-15)
 
+    def test_embedding_solver(self):
+        DVGeo = DVGeometry(os.path.join(self.base_path, "../../input_files/fuselage_ffd_severe.xyz"))
+
+        # none of these fail yet...
```

Comment:
What does this mean?

Reply:
that means I forgot to remove my old comments from the test :) my previous test point was working with the embedding fine. now they do fail with the old pyspline projection code